### PR TITLE
[Robot] Verify course connections health check

### PR DESCRIPTION
**Pull request by rhakeshj**
Added test to verify course connections health check in EDA "Settings Health Check" app and organized test cases based on individual health check cards

WI - 8976999

# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: robot/EDA/tests/browser/health_check/account_model_check.robot [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    Go to settings health check
+    Current page should be      Home        Settings Health Check
+    Run account model health check settings
+
+*** Keywords ***
+Run account model health check settings
+    [Documentation]             Validates the account model health check settings row by row
+    Click health check button       Run Health Check
+    ${all_checks_status} =      Return all checks status        Account Model       All checks passed
+    Log To Console              ${all_checks_status}
```
Comment (skristem) [inline robot/EDA/tests/browser/health_check/account_model_check.robot:25]: looks like u forgot to remove this

File: robot/EDA/tests/browser/health_check/account_model_check.robot [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    [Documentation]         Validates the account model health card is displayed with settings for
+    ...                     Administrative, Household, Default Account Model and Unique account
+    ...                     record type checks.Also verifies the status of each setting.
+    [tags]                  unstable        rbt:high        W-8880346
+    Go to settings health check
+    Current page should be      Home        Settings Health Check
+    Run account model health check settings
+
+*** Keywords ***
+Run account model health check settings
```
Comment (skristem) [inline robot/EDA/tests/browser/health_check/account_model_check.robot:21]: this keyword is doing the same for an account, affiliation, course connection. Create a keyword with name and dictionary for validation as variables and you can reuse the same keyword in all tests

File: robot/EDA/tests/browser/health_check/affiliation_mappings_check.robot [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    Go to settings health check
+    Current page should be      Home        Settings Health Check
+    Run affiliation mapping health check settings
+
+*** Keywords ***
+Run affiliation mapping health check settings
+    [Documentation]             Validates the affiliation mapping health check settings row by row
+    Click health check button       Run Health Check
+    ${all_checks_status} =      Return all checks status        Affiliation Mappings       All checks passed
+    Log To Console       ${all_checks_status}
```
Comment (skristem) [inline robot/EDA/tests/browser/health_check/affiliation_mappings_check.robot:25]: please remove this

File: robot/EDA/tests/browser/health_check/course_connections_check.robot [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    Go to settings health check
+    Current page should be      Home        Settings Health Check
+    Run course connections health check settings to check warning message
+
+*** Keywords ***
+Run course connections health check settings to check warning message
+    [Documentation]             Validates the course connections health check settings row by row
+    Click health check button       Run Health Check
+    ${all_checks_status} =      Return all checks status        Course Connections       All checks passed
+    Log To Console              ${all_checks_status}
```
Comment (skristem) [inline robot/EDA/tests/browser/health_check/course_connections_check.robot:28]: please remove this

File: robot/EDA/tests/browser/health_check/account_model_check.robot (hunk 1 above)
Comment (rhakeshj) [reply robot/EDA/tests/browser/health_check/account_model_check.robot]: This is removed now

File: robot/EDA/tests/browser/health_check/account_model_check.robot (hunk 2 above)
Comment (rhakeshj) [reply robot/EDA/tests/browser/health_check/account_model_check.robot]: Thanks for the suggestion. I made the change and customized to reuse it for all scenarios.

File: robot/EDA/tests/browser/health_check/affiliation_mappings_check.robot (hunk 3 above)
Comment (rhakeshj) [reply robot/EDA/tests/browser/health_check/affiliation_mappings_check.robot]: Done

File: robot/EDA/tests/browser/health_check/course_connections_check.robot (hunk 4 above)
Comment (rhakeshj) [reply robot/EDA/tests/browser/health_check/course_connections_check.robot]: Done

**Review by rjanjanam — APPROVED** [review]

**Review by spinkelman — APPROVED** [review]

**Review by skristem — APPROVED** [review]
looks good

**Pull request merged**